macrosAkka/src/main/scala/com/softwaremill/macwire/macwireakka/MacwireAkkaMacros.scala

```diff
@@ -0,0 +1,14 @@
+package com.softwaremill.macwire
+package macwireakka
```

Comment:
I've intentionally named it differently than 'akka' in order to bypass the problem when you need to uglyfy _akka_ imports with `_root_` prefix when you import it after importing `com.softwaremill.macwire.akka`.

Consider such code:

```scala
package whatever

import com.softwaremill.macwire._
import com.softwaremill.macwire.akka._ 

import akka.actor.ActorSystem

object Demo extends App {

  val system = ActorSystem("Demo")
//...
}

//Error:(8, 13) object actor is not a member of package com.softwaremill.macwire.akka
//import akka.actor.ActorSystem

//Error:(10, 16) not found: value ActorSystem
//  val system = ActorSystem("Demo")

```

This will not compile because compiler will look for `ActorSystem` in macwire. For begginers it can be confusing. Especially when users rely on IDE support and for example try to fix imports from Intellij which will not resolve it properly.

Of course we can simply fix it by adding `import akka.actor.ActorSystem` in front _macwire_ or `import _root_.akka.actor.ActorSystem` somewhere after _macwire_.

I was considering as well placing everything in `com.softwaremill.macwireakka` but in such scenario private stuff in `com.softwaremill.macwire` would not be accessible in this project.

What do you think? Shall we rename it?